x/concentrated-liquidity/spread_rewards.go

```diff
@@ -174,6 +174,11 @@ func (k Keeper) collectSpreadRewards(ctx sdk.Context, sender sdk.AccAddress, pos
 		return sdk.Coins{}, err
 	}
 
+	// Early return, emit no events if there is no spread rewards to claim.
+	if spreadRewardsClaimed.IsZero() {
+		return sdk.Coins{}, nil
+	}
+
```

Comment:
Slightly unrelated to this PR, but I think the change makes sense. I realized we would emit a bank send event coming from collecting spread rewards on every call, even when there are no spread rewards to claim. This logic added is similar to the incentives claiming logic that ends early if there are no incentives to claim.